lms/djangoapps/support/serializers.py

```diff
@@ -86,17 +86,36 @@ def serialize_user_info(user, user_social_auths=None):
     return user_info
 
 
-def serialize_sso_records(user_social_auths):
+def serialize_sso_records(user_social_auths, user_social_auths_history):
     """
     Serialize user social auth model object
     """
     sso_records = []
     for user_social_auth in user_social_auths:
+        history = serialize_sso_history(
+            user_social_auth,
+            user_social_auths_history
+        )
         sso_records.append({
             'provider': user_social_auth.provider,
             'uid': user_social_auth.uid,
             'created': user_social_auth.created,
             'modified': user_social_auth.modified,
             'extraData': json.dumps(user_social_auth.extra_data),
+            'history': history,
         })
     return sso_records
+
+
+def serialize_sso_history(user_social_auth, user_social_auths_history):
```

Comment:
user_social_auth is a list of auth object dicts, not a single object. How come this code is working?
```
user_social_auths = UserSocialAuth.objects.filter(user=user)
```